### PR TITLE
Do not use popover for license description 

### PR DESCRIPTION
**Pull request by christianmemije**
# Details

### Summary

* ui-popover positioning/sizing is unreliable on smaller viewports
* Instead just use a simple toggle.
* Another improvement for smaller devices

#### Before
before

#### After
after

### Reviewer guidance

* Navigate to a piece of content with a license description

### References

# Contributor Checklist

- [x] PR has the correct target milestone
- [x] PR has the appropriate labels
- [x] Changes in the PR do not introduce accessibility regressions ([confimed by testing with one of the recommended tools](http://kolibri.readthedocs.io/en/develop/dev/manual_testing.html#accessibility-a11y-testing)) 
- [x] If PR is ready for review, it has been assigned or requests review from someone
- [x] ~Documentation is updated as necessary~ N/A
- [x] ~External dependency files are updated (`yarn` and `pip`)~ N/A
- [x] ~If internal dependency is updated, link to diff is included~ N/A
- [x] Screenshots of any front-end changes are in the PR description
- [x] ~CHANGELOG.rst is updated for high-level changes~ N/A
- [x] ~You've added yourself to AUTHORS.rst if you're not there~

# Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] PR has been fully tested manually

### CODE REVIEW

**Comment by indirectlylit** [comment]
let's change the label from

```
licenseDescription: 'License description'
```

to 

```
licenseDescription: 'Toggle license'
```

File: kolibri/plugins/learn/assets/src/views/content-page/index.vue [hunk 1]
```diff
@@ -55,13 +55,16 @@
...
-            <ui-icon icon="info_outline" :ariaLabel="$tr('licenseDescription')" class="license-tooltip"/>
-          </span>
-
-          <ui-popover trigger="licensetooltip" class="license-description">
+          <ui-icon-button
+            :icon="showLicenceDescription ? 'expand_less' : 'expand_more'"
+            :ariaLabel="$tr('licenseDescription')"
+            size="small"
+            type="secondary"
+            @click="showLicenceDescription=!showLicenceDescription"
```
Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/views/content-page/index.vue]: `showLicenceDescription = !showLicenceDescription`

File: kolibri/plugins/learn/assets/src/views/content-page/index.vue [hunk 2]
```diff
@@ -135,7 +137,7 @@
       licenseDescription: 'License description',
       copyrightHolder: 'Copyright holder: {copyrightHolder}',
     },
-    data: () => ({ wasIncomplete: false }),
+    data: () => ({ wasIncomplete: false, showLicenceDescription: false }),
```
Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/views/content-page/index.vue]: minor, but `showLicenceDescription` sounds like a method name. Sometimes I like to start booleans with `is`, like `isLicenseVisible`?

File: kolibri/plugins/learn/assets/src/views/content-page/index.vue [hunk 3]
```diff
@@ -120,10 +123,8 @@
   import assessmentWrapper from '../assessment-wrapper';
   import pointsPopup from '../points-popup';
   import pointsSlidein from '../points-slidein';
-  import uiPopover from 'keen-ui/src/UiPopover';
-  import uiIcon from 'keen-ui/src/UiIcon';
+  import uiIconButton from 'keen-ui/src/UiIconButton';
   import markdownIt from 'markdown-it';
-  import { ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
```
Comment (christianmemije) [inline kolibri/plugins/learn/assets/src/views/content-page/index.vue:126]: Duplicate import

**Comment by christianmemije** [comment]
@indirectlylit Addressed comments.

**Pull request merged**